chat-ui/src/services/conversations.ts

```diff
@@ -59,10 +51,17 @@ export class TimeoutError<Data extends object = object> extends Error {
   }
 }
 
+type ConversationServiceConfig = {
+  serverUrl: string;
+};
+
 export class ConversationService {
   private serverUrl: string;
 
   constructor(config: ConversationServiceConfig) {
+    if(!config.serverUrl) {
```

Comment:
Prettier malfunction?
```suggestion
    if (!config.serverUrl) {
```